src/components/organisms/HomeIntro.tsx

```diff
@@ -58,41 +48,34 @@ interface HomeIntroData {
       }
     }
   }
-  waves: { childImageSharp: { fluid: FluidObject } }
 }
 
 export default function HomeIntro(): ReactElement {
   const data: HomeIntroData = useStaticQuery(query)
   const { teaser } = data.file.childIndexJson.intro
-  const waves = data.waves.childImageSharp.fluid
 
   return (
     <div className={styles.introWrapper}>
-      <Container>
-        <div className={styles.intro}>
-          <div className={styles.playButtonWrapper}>
-            <a
-              href="https://academy.delta-dao.com/"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              <div className={styles.playButton}>
-                <div className={styles.circle} />
-              </div>
-              <h2 className={styles.circleText}>start now</h2>
-              <Play className={styles.play} />
-            </a>
-          </div>
-          <div className={styles.waves}>
-            <Img fluid={waves} />
-          </div>
-          <div className={styles.education}>
-            <h4>{teaser.pre}</h4>
-            <h2>{teaser.title}</h2>
-            <Markdown text={teaser.text} />
+      <div className={styles.intro}>
+        <div className={styles.playButtonWrapper}>
+          <div className={styles.playButton}>
+            <div className={styles.circle}>
+              <a
+                href="https://academy.delta-dao.com/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <h2 className={styles.circleText}>start now</h2>
+              </a>
+            </div>
           </div>
```

Comment:
thought(non-blocking): looks like something that we could extract to a component.